app/services/container_dashboard_service.rb

```diff
@@ -175,8 +173,54 @@ def ems_utilization
           :yData => ["used"] + used_mem.values.map { |m| (m / 1024.0).round }
         }
       }
-    else
-      {}
     end
   end
+
+  def hourly_network_trends
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    hourly_network_trend = Hash.new(0)
+    hourly_metrics =
+      MetricRollup.find_all_by_interval_and_time_range("hourly", 24.hours.ago.beginning_of_hour.utc, Time.now.utc)
```

Comment:
nit: @zeari I think that `1.day.ago.beginning_of_hour.utc` would do as well (no need to change this if you prefer `24.hours`)